x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/planner/PlannerSettings.java

```diff
@@ -19,7 +19,7 @@
 /**
  * Values for cluster level settings used in physical planning.
  */
-public class PhysicalSettings {
+public class PlannerSettings {
```

Comment:
Ah. I was expecting a `LogicalSettings` on day. but this is just as good.

Reply:
I was a bit hesitant about this rename, but I thought it might make sense since there's no hard distinguishing between them so far and even the new one is actually using while executing (subplans). But yes, then can be untangled if needed.